packages/tron/src/events/index.ts

```diff
@@ -116,3 +131,14 @@ export default class Event {
         });
     }
 }
+
+function chunkArray(arr: string[], size: number) {
+    const result = [];
+
+    for (let index = 0; arr.length > size; index++) {
```

Comment:
Use While instead of For loop.
Variable index is never used  and the number of iterations is conditional.